### PR TITLE
netease-music-tui: 0.1.2 -> 0.1.3

### PR DESCRIPTION
**Pull request by Vonfry**
Add `update-cargo-lock.sh` as well.

###### Motivation for this change
Update version.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/netease-music-tui/update-cargo-lock.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/netease-music-tui/update-cargo-lock.sh:1]: This file should use nix-shell with coreutils, gnugrep, git and cargo.

File: pkgs/applications/audio/netease-music-tui/default.nix [hunk 2]
```diff
@@ -2,21 +2,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "netease-music-tui";
-  version = "v0.1.2";
+  version = "v0.1.3";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/netease-music-tui/default.nix:5]:
```suggestion
  version = "0.1.3";
```

File: pkgs/applications/audio/netease-music-tui/default.nix [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
 
   src = fetchFromGitHub {
     owner = "betta-cyber";
     repo = "netease-music-tui";
     rev = version;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/netease-music-tui/default.nix:10]:
```suggestion
    rev = "v${version}";
```

File: pkgs/applications/audio/netease-music-tui/default.nix [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
+    sha256 = "09355a6d197ckayh9833y39dsarklgpgrq3raapiv25z59di30qq";
   };
 
   cargoPatches = [ ./cargo-lock.patch ];
 
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ alsaLib openssl ];
 
-  cargoSha256 = "1kfbnwy3lkbhz0ggxwr5n6qd1plipkr1ycr3z2r7c0amrzzbkc7l";
+  cargoSha256 = "1p6z7p4z5kmsf12yfhxf84zzvkfsx0f1wl6mq5h0bsasdcba6v7s";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/netease-music-tui/default.nix:19]:
```suggestion
  cargoSha256 = "sha256:0f06wc7h2zjipifvxsskxvihjf6mykrjrm7yk0zf98ra079bc9g9";
```

**Comment by Vonfry** [comment]
Should I do something else, such as rebase these commit?

**Comment by SuperSandro2000** [comment]
> Should I do something else, such as rebase these commit?

Not required. I just squash merge it.

**Pull request merged**